### PR TITLE
Problem: we need new MB request to provide hw capability data

### PR DESCRIPTION
**Pull request by bstepankova**
Solution: added

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -134,6 +134,26 @@ where
...
+
+Reply:
+
+* 'msg-correlation-id'/OK/'type'/'count'/'base_address'/'offset'/'mapping1'/'mapping_val1'/'mapping2'/'mapping_val2'/ ...
+* 'msg-correlation-id'/ERROR/'reason'
+
+where:
+'count' - number of GPI/GPO pins
+'offset' - offset of pin numbering (GPI pins have -1 offset, i.e. GPI 1 is pin 0, ... )
+'maping' - Mapping between GPI/GPO number and HW pin number
```
Comment (aquette) [inline README.md]: typo: missing 'p'

Comment (bstepankova) [reply README.md]: fixed, thx

File: README.md [hunk 2]
```diff
@@ -134,6 +134,26 @@ where
 
     Value associated with ANY key MAY be NULL.
 
+#### HW Capability Request
+Request:
```
Comment (aquette) [inline README.md]: should follow the same structure and wording as "RC information".
Also add "HW Capability Request" to the supported "Mailbox requests"

File: src/fty_info.cc [hunk 3]
```diff
@@ -136,6 +137,7 @@ int main (int argc, char *argv [])
     }
 
     zstr_sendx (server, "PATH", path, NULL);
+    zstr_sendx (server, "GONFIG", hw_cap_path, NULL);
```
Comment (aquette) [inline src/fty_info.cc]: typo: you probably meant *C*ONFIG ;)

File: src/fty_info_server.cc [hunk 4]
```diff
@@ -37,6 +37,9 @@
 
 #include "fty_info_classes.h"
 
+# define OVA_HW_CAP "ova-capabilities.dsc"
```
Comment (aquette) [inline src/fty_info_server.cc]: you should only try to access the generic file 42ity-capabilities.dsc

File: README.md [hunk 5]
```diff
@@ -134,6 +135,25 @@ where
 
     Value associated with ANY key MAY be NULL.
 
+#### HW Capability Request
+
+* HW_CAP/'msg-correlation-id'/'type'
+
+where:
+
+* 'type' can be GPI, GPO, serial (not supported yet)
```
Comment (aquette) [inline README.md]: is this case insensitive? otherwise 'gpi' and 'gpo' are more suitable

Comment (bstepankova) [reply README.md]: well, fty-sensor-gpio uses capital letters too

Comment (bstepankova) [reply README.md]: ok, this is part of the the message it should not be capital. Corrected

**Pull request merged**